PWGCF/FEMTOSCOPY/FemtoDream/AliFemtoDreamCorrHists.h

```diff
@@ -99,8 +99,15 @@ class AliFemtoDreamCorrHists {
     if(fMassQADistPart1[i] && fMassQADistPart2[i]) {
       fMassQADistPart1[i]->Fill(invMass1, kstar);
       fMassQADistPart2[i]->Fill(invMass2, kstar);
+      //fPairInvMassQAD[i]->Fill(invMass1, invMass2, kstar); //?????????      
     }
-  }
+
+}
+  void FillPairInvMassQAD(int i, Double_t invMass) {
```

Comment:
maybe do the calculation of the inv mass here?
void FillPairInvMassQAD(int i, AliFemtoDreamBasePart *part1, AliFemtoDreamBasePart *part2) {
  .. TLorentzVector stuff
}